deploy/charts/istio-csr/values.yaml

```diff
@@ -298,3 +298,14 @@ nodeSelector:
 
 # Labels to apply to all resources
 commonLabels: {}
+
+# Create dynamic manifests via values.
+#
+# For example:
+# extraObjects:
+#   - |
+#     apiVersion: v1
+#     kind: ConfigMap
+#     metadata:
+#       name: '{{ template "cert-manager.fullname" . }}-extra-configmap'
```

Comment:
suggestion: This example doesn't actually work locally for me because `cert-manager.fullname` doesn't seem to be defined. It's minor, but I think the example would be better without the template bit

```suggestion
# For example:
# extraObjects:
#   - |
#     apiVersion: v1
#     kind: ConfigMap
#     metadata:
#       name: 'some-extra-configmap'
```